### PR TITLE
 changed scrapping logic so that it provides user with 2 choices to e…

### PR DESCRIPTION
**Pull request by NkwaTambe**
Enhances scrapping logic to allow users to choose between scraping text or images. Additionally, calculates the total number of characters for scraped text.

### CODE REVIEW

File: scraping.py [hunk 1]
```diff
@@ -1,26 +1,36 @@
...
+        if choice == '1':  # User chose to scrape text
+            for element in soup.find_all('p'):
+                extracted_data.append(element.text)
+                total_characters += len(element.text)
+        elif choice == '2':  # User chose to scrape images
+            for img in soup.find_all('img', src=True):
+                extracted_data.append(img['src'])
+        else:
+            print("Invalid choice. Please enter '1' or '2'.")
+            return None
```
Comment (stephane-segning) [inline scraping.py:24]: User don't choose either text or images, but both always.

**Review by stephane-segning — APPROVED** [review]
Good code @NkwaTambe. You see those `…` in the title? It's because it's too long. Try changing it the way I showed you and reduce it to the minimum, using the conventions I showed you [here](https://www.conventionalcommits.org/en/v1.0.0/). Also, update this file here:

File: scraping.py (hunk 1 above)
Comment (NkwaTambe) [reply scraping.py:24]: ok sir @stephane-segning .
adjusting changes now as requested

**Comment by NkwaTambe** [comment]
hello sir @stephane-segning , i have done the changes , but looks like i can't create another pull request, please help me verify the changes if its  correct

**Pull request merged**